### DISCUSSION TITLE
Volume boost option

### DISCUSSION THREAD

**Discussion by melorin83** [AntonyCorbett/OnlyM#420] [Ideas]
Hello!
It would be nice to have a volume boost option, because when playing videos/kingdom melodies and using share screen function in Zoom, the volume is lower than normal somehow. Could be a button on the skin to On-Off the boost and in the options, we can set the dB value to our liking.
Thank you very much for your efforts so far!
Cheers!

**Comment by AntonyCorbett** [comment]
@melorin83 Thanks for raising the subject. Please search on "Volume" in the Issues section as I think the subject is covered there.

**Reply by melorin83** [reply]
I have read other issues about volume problems. I don't want to install more 3rd party software on Windows to boost system volume. Also, Windows built in 'Loudness compensation' function works in a strange behavior, which is boost volume, but Zoom still lowers the shared sound.
Would be better to have such an option directly in the movie player, in this case OnlyM.

**Comment by cccinfo** [comment]
I concur with @melorin83. This would be a very useful feature. Frequently, we must play foreign-language vids recorded in various locations for meeting parts. Boosting video volume via OnlyM is practical and useful. External progs is a bit of a kludge.

A related issue is audio/vid out of sync. But that is for another "issue" requiring your great minds, eh?

Thank you for your hard work.
